fastlane/lib/assets/ActionDetails.md.erb

```diff
@@ -51,6 +51,21 @@ Key | Description | Default
 
 <hr />
 
+<% if action.output && action.output.kind_of?(Array) && action.output.length > 0 %>
+
+## Outputs
+
+Shared Area Key | Description 
+----------------|-------------
+<%- (action.output || []).each do |array_item| -%>
+  <%- next unless array_item.kind_of?(Array) -%>
+  <%- next if array_item.length != 2 -%>
+  `<%= array_item[0] %>` | <%= array_item[1] %>
```

Comment:
We could even think about prefixing this with `SharedValues::` here so it can just be copy pasted into `lane_context[...]` to be used. It repeats a few times, but probably makes it clearer how it should be used (especially if the accompanying description text also includes a generalized example how to use `lane_context`).

Reply:
Good idea, fixed 💪